pkg/plugin/trivy/plugin.go

```diff
@@ -19,19 +19,6 @@ import (
 	"k8s.io/utils/pointer"
 )
 
-var (
```

Comment:
For backward compatibility we should add requests and limits to the Init callback, which is used to ensure the default config for a plugin:

```go
// Init ensures the default Config required by this plugin.
func (p *plugin) Init(ctx starboard.PluginContext) error {
	return ctx.EnsureConfig(starboard.PluginConfig{
		Data: map[string]string{
			keyTrivyImageRef: "docker.io/aquasec/trivy:0.16.0",
			keyTrivySeverity: "UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL",
			keyTrivyMode:     string(Standalone),

			"trivy.resources.request.cpu":    "100m",
			"trivy.resources.request.memory": "100M",
			"trivy.resources.limit.cpu":      "500m",
			"trivy.resources.limit.memory":   "500M",
		},
	})
}
```

It is also used by Starboard CLI in pkg/cmd/installer.go